### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by lucien2712** [AlexeyAB/darknet#8206]
I've built the environment
but when i want to test the result, there is a error i dont know how to solve.
Plz help me, thanks

the error:

D:\yolo_v4\darknet\build\darknet\x64>darknet.exe detector test cfg/coco.data cfg/yolov4.cfg yolov4.weights test.mp4 -dont_show -ext_output
 CUDA-version: 11050 (11050), cuDNN: 8.3.0, GPU count: 1
 OpenCV version: 4.1.0
 0 : compute_capability = 610, cudnn_half = 0, GPU: NVIDIA GeForce MX250
net.optimized_memory = 0
mini_batch = 1, batch = 8, time_steps = 1, train = 0
   layer   filters  size/strd(dil)      input                output
   0 Create CUDA-stream - 0
Could not load library cudnn_cnn_infer64_8.dll. Error code 126
Please make sure cudnn_cnn_infer64_8.dll is in your library path!

**Comment by bielu** [comment]
@lucien2712 I hit same issue, downgrade cudnn to 8.2.4 that is bug in cudnn :)

**Comment by lucien2712** [comment]
> @lucien2712 I hit same issue, downgrade cudnn to 8.2.4 that is bug in cudnn :)

got it! i've solved. Tks!!

**Comment by RohitSingh1226** [comment]
Did you have to build again after replacing?
After you replaced the cudnn files into your C:/Nvidia/cuda/v11.xx 
I am also facing the same issue. I have cudnn v8.3.0

![image](https://user-images.githubusercontent.com/35935157/141280442-9c178364-740e-4997-9cf1-4215757398bc.png)
@lucien2712 @bielu any response will be appreciated

**Comment by RohitSingh1226** [comment]
Also cudnn: 8.2.4 is for Cuda10.2 
But me and @lucien2712 have v11.5
Will that be a problem?

**Comment by lucien2712** [comment]
@RohitSingh1226 Now I downgrade to cuda 11.2 and cudnn 8.1
U can try it

**Comment by RohitSingh1226** [comment]
What is the fps increment you are getting after GPU.

@lucien2712 

Basically I just copied the Cudnn 8.2.4 for cuda 10 to bin/include/lib files into my Nvidia/GPu/cudav11.5 bin/include/lib

@AlexeyAB 

I am getting the same FPS approximately
![gpu](https://user-images.githubusercontent.com/35935157/141291648-e928b82d-2582-4844-b7f2-fc72e9a2d0dc.PNG)
  My GPU usage is not more than 15%

**Comment by RohitSingh1226** [comment]
The average fps is still 10.4

C:\Users\*****\OneDrive\Desktop\player-detection\vcpkg\installed\x64-windows\tools\darknet>darknet.exe detector demo cfg/coco.data cfg/yolov4.cfg yolov4.weights trim.mp4 -thresh 0.25
 CUDA-version: 11050 (11050), cuDNN: 8.3.0, GPU count: 1
 OpenCV version: 4.5.3
Demo
 0 : compute_capability = 750, cudnn_half = 0, GPU: NVIDIA GeForce GTX 1650
net.optimized_memory = 0
mini_batch = 1, batch = 8, time_steps = 1, train = 0
   layer   filters  size/strd(dil)      input                output
   0 Create CUDA-stream - 0
 Create cudnn-handle 0

**Comment by lucien2712** [comment]
about this part, i copied the **data in cuda /(bin/include/lib)** to  Nvidia/GPu/cudav11.5 bin/include/lib 
**rather than replacing**

> What is the fps increment you are getting after GPU.
> 
> @lucien2712
> 
> Basically I just copied the Cudnn 8.2.4 for cuda 10 to bin/include/lib files into my Nvidia/GPu/cudav11.5 bin/include/lib
> 
> @AlexeyAB
> 
> I am getting the same FPS approximately ![gpu](https://user-images.githubusercontent.com/35935157/141291648-e928b82d-2582-4844-b7f2-fc72e9a2d0dc.PNG) My GPU usage is not more than 15%

**Comment by RohitSingh1226** [comment]
What fps are you getting @lucien2712 in your case using the GPU?

**Comment by RohitSingh1226** [comment]
> @RohitSingh1226 Now I downgrade to cuda 11.2 and cudnn 8.1 U can try it

Did this work for you?

**Comment by lucien2712** [comment]
yes, but my gpu is trash mx250, so i only get  about 3.5 fps  @RohitSingh1226

**Comment by lucien2712** [comment]
@RohitSingh1226  but i think u r successful if u build the gpu environment with the steps. and there is no error message when u test
